pkg/testfixtures/storage/mysql.go

```diff
@@ -47,87 +46,32 @@ func (m *mySQLTestContainer) GetDatabaseSchemaVersion() int64 {
 // bootstrapped implementation of the DatastoreTestContainer interface wired up for the
 // MySQL datastore engine.
 func (m *mySQLTestContainer) RunMySQLTestContainer(t testing.TB) DatastoreTestContainer {
-	dockerClient, err := client.NewClientWithOpts(
-		client.FromEnv,
-		client.WithAPIVersionNegotiation(),
+	ctx := context.Background()
+
+	mysqlContainer, err := testcontainersmysql.RunContainer(ctx,
+		testcontainers.WithImage(mySQLImage),
+		testcontainers.WithEnv(map[string]string{
+			"MYSQL_DATABASE":      "defaultdb",
+			"MYSQL_ROOT_PASSWORD": "secret",
+		}),
+		testcontainers.WithWaitStrategy(wait.ForLog("port: 3306  MySQL Community Server - GPL")),
```

Comment:
nitpick (non-blocking): Instead of hard-coding 3306 (mysql default port) in here and line 70, perhaps use set 3306 as a constant value so that (if) we update the port, the log will not fell out of date?

Reply:
I know that GPL in there is gonna trigger FOSSA :)

What does it mean?

Reply:
@adriantam yes 3306 is default port, I think there is no situation that we will change it in test containers, if we do then this env will be set too: `MYSQL_TCP_PORT`, keeping this as hard-coded string instead of making constant and string interpolation looks good to me.

@rhamzeh not sure what you meant, the log I got from testcontainers-go module example

Reply:
The module already comes with a default wait strategy which is almost the same as this one:

```
WaitingFor: wait.ForLog("port: 3306  MySQL Community Server"),
```

Reply:
yeah, I have removed duplicate waiting strategy